akka-typed/src/main/scala/akka/typed/persistence/scaladsl/PersistentActor.scala

```diff
@@ -0,0 +1,140 @@
+/**
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.typed.persistence.scaladsl
+
+import akka.annotation.DoNotInherit
+import akka.annotation.InternalApi
+import akka.typed.Behavior.UntypedBehavior
+import akka.typed.Signal
+import akka.typed.persistence.internal.PersistentActorImpl
+import akka.typed.scaladsl.ActorContext
+
+object PersistentActor {
+  def persistent[Command, Event, State](
+    persistenceId: String,
+    initialState:  State,
+    actions:       Actions[Command, Event, State],
+    onEvent:       (Event, State) ⇒ State): PersistentBehavior[Command, Event, State] =
+    new PersistentBehavior(persistenceId, initialState, actions, onEvent,
+      recoveryCompleted = (state, _) ⇒ state)
+
+  sealed abstract class PersistentEffect[+Event, State]() {
```

Comment:
this should probably be named `Effect`?

Reply:
Works for me, the package gives a hint anyway